.github/workflows/main_workflow.yml

```diff
@@ -187,7 +187,7 @@ jobs:
           sudo mv tendermint /usr/local/bin/tendermint
 
           # install skaffold
-          curl -Lo skaffold https://storage.googleapis.com/skaffold/releases/v1.33.0/skaffold-linux-amd64 && \
+          curl -Lo skaffold https://storage.googleapis.com/skaffold/releases/v1.39.0/skaffold-linux-amd64 && \
```

Comment:
Can we take this to v1.39.1

Reply:
Why?

Reply:
This issue here;

https://github.com/GoogleContainerTools/skaffold/issues/7587

I checked my skaffold version and I have already bumped to latest and resolved it

![image](https://user-images.githubusercontent.com/35799987/181915096-ed8359c4-3887-4dec-bd08-a337c2cda04f.png)